domains_plugin.php

```diff
@@ -883,6 +883,19 @@ public function getActions()
                 'name' => 'DomainsPlugin.nav_secondary_staff.domains',
                 'options' => ['parent' => 'billing/']
             ],
+            // Domains Client Nav
+            [
```

Comment:
I'm not a huge fan of this subitem being generated by the domain manager.  Instead, let's add it to the core and update the client structure.pdt to mimic the admin structure.pdt in hiding the nav dropdown if there is only one subnav item and it matches the parent.